Jenkinsfile

```diff
@@ -3672,7 +3672,7 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         model.decoder.bias_activation_fusion=False \
         model.decoder.activations_checkpoint_method='block' \
         model.decoder.activations_checkpoint_num_layers=1 \
-        model.data.data_prefix=[.5,/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document,.5,/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document]"
+        model.data.data_prefix='{train:[1.0,/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document],test:[/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document], validation:[/home/TestData/nlp/megatron_t5/data/pile_val_small_bert_tokenizer_text_document]}'"
```

Comment:
I don't think we want to change this test. We need to make sure we stay backward compatible.

Reply:
I made this recommendation to @shanmugamr1992 that we re-purpose one of our many tests (BART/T5/UL2 w/ TP=2,PP=2 combinations) to test this feature rather than adding a new CI test and increasing our CI time. Things are still backward compatible and you can provide things the old way as well.